app/models/distribution.rb

```diff
@@ -133,4 +133,8 @@ def csv_export_attributes
       line_items.total
     ]
   end
+
+  def can_be_edited?
```

Comment:
I think the name here is a little deceptive, as a distribution can always be edited by an admin. Can we change it to something like `past?`

Reply:
Resolved